spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -487,6 +487,8 @@ def when_I_unschedule_publishing
   end
 
   def and_the_step_by_step_is_not_editable
+    there_should_be_no_reorder_steps_tab
```

Comment:
I think this should be renamed to `then...` as "there" isn't one of the step keywords for cucumber tests: https://cucumber.io/docs/gherkin/reference/#keywords